userspace/falco/output.proto

```diff
@@ -0,0 +1,28 @@
+syntax = "proto3";
+
+import "google/protobuf/timestamp.proto";
+import "schema.proto";
+
+package falco.output;
+
+option go_package = "github.com/falcosecurity/client-go/pkg/api/output";
+
+service service {
+  rpc subscribe(request) returns (stream response);
```

Comment:
These names are pretty generic, can you change them to something like subscribe_alerts so it's clear that the client is subscribing to all falco alerts, and not other streams of info?

Reply:
Yes but they are namespaced. Thus being scoped to a namespace they at the end result in pretty significant name imho, eg., `falco::output::service`.